shared/store/action-logger.js

```diff
@@ -39,9 +38,7 @@ const actionStatSink: StatSink = {
 }
 
 export const actionLogger = (store: any) => (next: any) => (action: any) => {
-  console.groupCollapsed && console.groupCollapsed(`Dispatching action: ${action.type}`)
-
-  console.log(`Dispatching action: ${action.type}: `, forwardLogs ? JSON.stringify(action) : action)
+  const log1 = `Dispatching action: ${action.type}: ${forwardLogs ? JSON.stringify(action) : action}`
```

Comment:
This won't work. It'll become `[Object object]` (when not stringified). this should be an array that gets applied later.